test/workitemrepository.go

```diff
@@ -0,0 +1,277 @@
+// This file was generated by counterfeiter
```

Comment:
A little doc somewhere that describes how to regenerate this when it changes would be helpful.